test/chrono-test.cc

```diff
@@ -287,19 +287,48 @@ TEST(chrono_test, system_clock_time_point) {
     EXPECT_EQ(sys_output, fmt::format(fmt::runtime(fmt_spec), tm));
   }
 
-  if (std::find(spec_list.cbegin(), spec_list.cend(), "%z") !=
-      spec_list.cend()) {
+#if defined(__MINGW32__) && !defined(_UCRT)
+  spec_list = {"%Z"};
+#else
+  spec_list = {"%z", "%Z"};
+#endif
```

Comment:
I suggest moving this code to a separate function and reuse here and elsewhere in chrono-test.

Reply:
A comment clarifying why these are handled separately would be nice too.

Reply:
For loops are not equivalent.

First ``for`` loop tests ``std::chrono::time_point`` (``std::chrono::system_clock::now()``) and ``std::tm`` (in UTC).
Second ``for`` loop tests timezone formatters only for ``std::tm`` (in localtime).

I think we should not do this, since timezone support is a weak point of fmt and the standard library.

Reply:
@vitaut 
Comments added.

Reply:
I didn't mean merging the for loops, just moving the selected block, namely

```c++
#if defined(__MINGW32__) && !defined(_UCRT)
  spec_list = {"%Z"};
#else
  spec_list = {"%z", "%Z"};
#endif
```

into a separate function (say, `get_timezone_specs`) to keep the system-specific conditional compilation in one place.

Other than that, all looks good.

Reply:
@vitaut
I don't understand how this will simplify the code.
The condition ``defined(__MINGW32__) && !defined(_UCRT)`` is used 4 times in this test. Partially with precondition ``ifndef _WIN32``.

Reply:
It is not perfect but would be slightly cleaner because we could reuse this function in two of these places. Another option is to move the `defined(__MINGW32__) && !defined(_UCRT)` condition into one place, define our own macro (e.g. `FMT_HAS_C99_STRFTIME`) and reuse that in all places.

Reply:
Done.
I added macro ``FMT_HAS_C99_STRFTIME``.